extension/observer/ecstaskobserver/config.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package ecstaskobserver // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/ecstaskobserver"
+package ecstaskobserver // import "github.com/open-telemetry/opentelemetry-collector-contrib/extensionImpl/observer/ecstaskobserver"
```

Comment:
Curios how this change happended here? :)

Reply:
Fixed now.